unittests/api_tests.cpp

```diff
@@ -2372,6 +2372,38 @@ BOOST_FIXTURE_TEST_CASE(action_ordinal_test, TESTER) { try {
    set_code( N(erin), contracts::test_api_wasm() );
    produce_blocks(1);
 
+   // generate test hash
+   auto gth = [&](auto ac) {
+      action testact;
+      testact.account = ac.account;
+      testact.name    = ac.name;
+      testact.authorization = ac.authorization;
+      std::vector<char> buff(fc::raw::pack_size(testact));
```

Comment:
Why copy into `testact` when you can just serialize the `action_base` portion of `ac`? I think it would also make this code a little easier to follow.